Helper/DbConnectionParamsNormalizer.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of TheCadien/SuluImportExportBundle.
+ *
+ * (c) Oliver Kossin
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace TheCadien\Bundle\SuluImportExportBundle\Helper;
+
+class DbConnectionParamsNormalizer
```

Comment:
Follow up to my other comment, this could be used as a factory in the SuluImportExportExtension to provide the Configuration as a service to the Import-/ExportService. That would keep the Doctrine config out of our services, hopefully making upgrades/compatibility with DoctrineBundle a bit easier by keeping them more contained.

Again, this can be done in a follow up as well, so no need to change this here, unless you are super eager to do it.